client-sdk/ts-web/core/src/misc.ts

```diff
@@ -1,4 +1,3 @@
-// @ts-expect-error missing declaration
```

Comment:
This was preventing npm i from generating a package-lock properly and ts-web/core had no package-lock file

Reply:
This was recently removed from main. Could you rebase?

Reply:
Done